### ISSUE TITLE
Additional text after image in imgcaption tag is displayed as caption

### ISSUE THREAD

**Issue by Klap-in** [heinrichheine/dokuwiki-imagereference#5]
The addEntryPattern() match till the | in 
`<imgcaption image1|Caption>Text1{{image.png}}Text2</caption>`

The `Caption>Text1` and `Text2` are both matched as DOKU_LEXER_UNMATCHED, so the action in DOKU_LEXER_UNMATCHED is called twice. The second time the correct caption founded in the first run is overwritten by `Text2`. 

Solution is rewriting the regexp of the addEntryPattern() so that the whole `<imgcaption image1|Caption>` is parsed in DOKU_LEXER_ENTER.

Inspiration available at:
https://www.dokuwiki.org/devel:plugin_survey:syntax#entry_exit_patterns

**Comment by Klap-in** [comment]
fixed in PR with  4e05d7a

**Comment by Klap-in** [comment]
this also fixes next bug reported in wiki:

```
The image captions do not work correctly when they contain acronyms recognized by DokuWiki. 
The acronyms get  pulled out and placed BEFORE the image tag in the HTML, and it renders badly.
```

When checking whether the caption text is unmatched it's recognized as acronym, and the old caption detection matches on the tags of the acronym instead of end tag of imgcaption
